src/entities/removeLiquidity/removeLiquidityV3/encodeRemoveLiquiditySingleTokenExactIn.ts

```diff
@@ -1,15 +1,18 @@
-import { encodeFunctionData, Hex } from 'viem';
-import { RemoveLiquidityBaseBuildCallInput } from '../types';
+import { encodeFunctionData } from 'viem';
+import { RemoveLiquidityV3BuildCallInput } from '../types';
 import { balancerRouterAbiExtended } from '@/abi';
+import { missingParameterError } from '@/utils';
 
 export const encodeRemoveLiquiditySingleTokenExactIn = (
-    input: RemoveLiquidityBaseBuildCallInput & { userData: Hex },
+    input: RemoveLiquidityV3BuildCallInput,
```

Comment:
lol @johngrantuk with the "**W**e **E**njoy **T**yping" practice 4 months ago :joy: